.github/release.yaml

```diff
@@ -0,0 +1,43 @@
+# Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+# Exceptions. See /LICENSE for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Provide the configuration and categorization for generated release notes.
+
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+
+  categories:
+    - title: Proposals Accepted and Merged 📜
+      labels:
+        - proposal
+      exclude.labels:
+        - automated
+    - title: Toolchain Changes 🛠
+      labels:
+        - toolchain
+      exclude.labels:
+        - automated
+    - title: Explorer Changes 🔭
+      labels:
+        - explorer
+      exclude.labels:
+        - automated
+    - title: Documentation Changes 🖺
```

Comment:
This unicode isn't rendering for me.

Had you considered using [GitHub's supported emoji](https://github-emoji-picker.vercel.app/), maybe avoiding rendering issues that way?  e.g.,`:scroll:` in place of the unicode. Just not sure whether GitHub would handle the replacement on generation.

`:hammer_and_wrench:` is what made me choose this particular link (others missed it)
:hammer_and_wrench:

Reply:
Sure, and thanks for the link to find all of these.